src/js/stores/__tests__/CosmosPackagesStore-test.js

```diff
@@ -200,6 +200,106 @@ describe("CosmosPackagesStore", function() {
     });
   });
 
+  describe("#fetchPackageVersions", function() {
+    const packageVersions = {
```

Comment:
#nit: You could put this in a fixture file.

Reply:
yes, the ideal in unit tests are to keep them on the file but because our mock data/fixtures are big and we use them for integration/system test.

And the more dependencies the worse.

that being said this mock being small I preferred to keep it only for the`describe` here.